dropshot/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "dropshot"
+description = "expose REST APIs from a Rust program"
+version = "0.1.0"
+authors = ["David Pacheco <dap@oxidecomputer.com>"]
+edition = "2018"
+license = "Apache-2.0"
+repository = "https://github.com/oxidecomputer/dropshot/"
+
+[dependencies]
+async-trait = "0.1.24"
+bytes = "0.5.4"
+chrono = { version = "0.4", features = [ "serde" ] }
+dropshot_endpoint = { version = "0.1.0", path = "../dropshot_endpoint" }
+futures = "0.3.1"
+http = "0.2.0"
+hostname = "0.3.0"
+hyper = "0.13"
+indexmap = "1"
+openapiv3 = "0.3"
+schemars = "0.7"
+serde = { version = "1.0", features = [ "derive" ] }
+serde_json = "1.0"
+serde_urlencoded = "0.6"
+slog-async = "2.4"
+slog-bunyan = "2.2"
+slog-json = "2.3"
+slog-term = "2.5"
+tokio = { version = "0.2", features = [ "full" ] }
+toml = "0.5.6"
+uuid = { version = "0.8", features = [ "serde", "v4" ] }
+
+[dependencies.slog]
```

Comment:
So, I prefer this style, but we should pick either one or the other. That is, this and, say, the UUID example above. They do the same things two different ways. Consistency is more important than which one is picked.

Reply:
Will look into this in a follow up (this was there before this change).

Reply:
https://github.com/oxidecomputer/dropshot/pull/3